.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,20 @@
----
-Make sure to mark the boxes below before creating PR: [x]
+<!--
+Thank you for contributing! Please make sure that your code changes
+are covered with tests. And in case of new features or big changes
+remember to adjust the documentation.
+
+Feel free to ping committers for the review!
 
-- [ ] Description above provides context of the change
-- [ ] Unit tests coverage for changes (not needed for documentation changes)
-- [ ] Target Github ISSUE in description if exists
-- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
-- [ ] Relevant documentation is updated including usage instructions.
-- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).
+In case of existing issue, reference it using one of the following:
+
+closes: #ISSUE
+related: #ISSUE
+-->
```

Comment:
I feel somewhere we should add a link to Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

Maybe even in comments or in Pull Request Guidelines too is fine :)

Reply:
I've added the link to the comment. It's also available in CONTRIBUTING.rst:
```
-   Adhere to guidelines for commit messages described in this `article <http://chris.beams.io/posts/git-commit/>`__.
    This makes the lives of those who come after you a lot easier.
```